presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -60,6 +60,7 @@ public class IcebergConfig
     private long maxManifestCacheSize = IO_MANIFEST_CACHE_MAX_TOTAL_BYTES_DEFAULT;
     private long manifestCacheExpireDuration = IO_MANIFEST_CACHE_EXPIRATION_INTERVAL_MS_DEFAULT;
     private long manifestCacheMaxContentLength = IO_MANIFEST_CACHE_MAX_CONTENT_LENGTH_DEFAULT;
+    private int splitManagerThreads = Runtime.getRuntime().availableProcessors();
```

Comment:
I noticed that they used `Runtime.getRuntime().availableProcessors() * 2` as the default value, probably because each processor core commonly supports 2 threads (hyper-threading). Are there any specific reasons for your setting?

Reply:
This default value is to be consistent with the default value of Iceberg's `iceberg.worker.num-threads` attribute：https://github.com/apache/iceberg/blob/main/core/src/main/java/org/apache/iceberg/SystemConfigs.java#L38-L43

Reply:
Gotcha. Thanks for the clarification.